woocommerce/woocommerce-ios
# Add UI Test for Orders List and Single Order Screen

Project: Critical Woo iOS UI Tests (pdcxQM-cS)

### Description
This PR adds a test for Orders, based on the structure and approach of the [Products](https://github.com/woocommerce/woocommerce-ios/pull/5674/) and [Reviews](https://github.com/woocommerce/woocommerce-ios/pull/5776) tests.
It should login using WordPress.com credentials, tap on Orders, check the Orders List, tap on the first Order, and check the Single Order that is loaded.

### Testing instructions
Pull the branch and run the test. 
Temporarily change some assertions within `OrdersScreen` or `SingleOrderScreen`, to make sure it fails when it should.
Temporarily change some data in `orders_any` to make sure it fails when it should.

Please suggest documentation for things that aren't clear!
